test/functional/gridfs_stream.test.js

```diff
@@ -1380,4 +1382,43 @@ describe('GridFS Stream', function() {
       // END
     }
   });
+
+  it('should correctly handle indexes created in shell', function(done) {
+    const configuration = this.configuration;
+    const url = format(
+      'mongodb://%s/%s',
+      format('%s:%s', configuration.host, configuration.port),
+      configuration.db
+    );
+    const cmd = `mongo ${url} --eval 'db.fs.files.createIndex({"filename" : 1,"uploadDate" : 1})'`;
+    child_process.exec(cmd, err => {
```

Comment:
How do we feel about this? Any alternatives? How can a test run command in the shell?

Reply:
I think you might be reading the ticket a little literally. If you look at the fix for [another driver](https://github.com/rozza/mongo-java-driver/pull/362/files) you'll see that this issue is about type mismatches when creating an index. We are likely not affected by this because node is loosely typed, but the root of the ticket is about the type mismatch and so coverage should be added around that.

IMO we should _never_ spawn child processes from our tests.